resources/functionMap.php

```diff
@@ -11804,7 +11804,7 @@
 'strtotime' => ['int|false', 'time'=>'string', 'now='=>'int'],
 'strtoupper' => ['string', 'str'=>'string'],
 'strtr' => ['string', 'str'=>'string', 'from'=>'string', 'to'=>'string'],
-'strtr\'1' => ['string', 'str'=>'string', 'replace_pairs'=>'array'],
+'strtr\'1' => ['string', 'str'=>'string', 'replace_pairs'=>'array<string,string>'],
```

Comment:
```suggestion
'strtr\'1' => ['string', 'str'=>'string', 'replace_pairs'=>'array<non-empty-string,string>'],
```

According to
> If replace_pairs contains a key which is an empty string (""), the element is ignored; as of PHP 8.0.0 E_WARNING is raised in this case.

https://www.php.net/manual/en/function.strtr.php